redash/handlers/query_results.py

```diff
@@ -171,8 +190,10 @@ def post(self, query_id):
         max_age = int(params.get('max_age', 0))
 
         query = get_object_or_404(models.Query.get_by_id_and_org, query_id, self.current_org)
+        parameter_schema = map(self._convert_queries_to_enums,
+                               query.options.get("parameters", {}))
```

Comment:
`parameters` is a list, so the default value should be `[]`, right? (I think this needs to be updated in other places in the code too)

Reply:
Good catch!